blog/2024-06-06-we-are-on-gpt-4o.mdx

```diff
@@ -0,0 +1,27 @@
+---
+slug: we-are-on-chatgpt-4o
+title: We are on GPT 4o 🚀
+description: Tolgee Translator is now backed by GPT-4o, the latest OpenAI's flagship model.
+authors: [jcizmar]
+tags: [chatgpt, tolgee, ai, tolgee-translator]
+---
+
+Great news! 🚀 Tolgee Translator is now backed by GPT-4o, the latest OpenAI's flagship model. It returns better results, and it is much less error-prone.
+
+<!--truncate-->
+
+## It fails less
+The GPT-3-turbo we used before sometimes failed since it could not provide results for some string language
+pairs. Due to this problem, sometimes the translation fails. The GPT 4o is more stable.
+
+## It understands the context better
+The GPT-4o better understands the context of the strings gathered from the Tolgee SDKs.
+So, it provides better results based on the context provided.
+Read more about the Tolgee Context [here](/js-sdk#automatic-context-collection).
+
+## It honors the AI Customization more
+If you provide a key description, project description or key description, the Translator will use this
+information to provide better results.
+In GPT-3-turbo, we have seen results where this information wasn't completely honored, probably due to
+the overall complexity of the prompt. For GPT-4o, the results are improved a lot. Read more about the AI Customization
```

Comment:
However, GPT-4o handles this complexity much better, ensuring that all provided context is taken into account. This leads to more accurate and tailored translations, perfectly aligned with your project's needs.